docs/pg-tde.md

```diff
@@ -194,4 +194,29 @@ To check if the data is encrypted, do the following:
     SELECT pg_tde_is_encrypted('my_encrypted_table');
     ```
 
-    The function returns `t` if the table is encrypted and `f` - if not.
\ No newline at end of file
+    The function returns `t` if the table is encrypted and `f` - if not.
+
+## Upgrade considerations
+
+Note that you cannot upgrade to `pg_tde` Beta with the UPDATE TO statement for the ALTER EXTENSION command. If you have previously installed `pg_tde` Alpha1 and wish to use the Beta version, you need to drop the extension and reenable the new version of it.
+
+<i warning>:material-alert: Warning:</i> Note that dropping the extension deletes all tables where it is enabled!
+
+To proceed with the migration to the new version, do the following:
+
+1. Drop the Alpha1 version of pg_tde on the desired database(s):
+
+    ```sql
+    DROP EXTENSION pg_tde
+    ```
+
+    <i warning>:material-alert: Warning:</i>Dropping the extension deletes all tables where it is enabled.
```

Comment:
It doesn't. If there are any pg_tde tables, this will report an error. The user has to drop pg_tde tables manually, or use `DROP EXTENSION pg_tde CASCADE`